### PR TITLE
set configsvr configuration option

### PR DESCRIPTION
**Pull request by jspiewak**
Addresses #218

### CODE REVIEW

File: recipes/configserver.rb [hunk 1]
```diff
@@ -23,9 +23,9 @@
 
 include_recipe "mongodb::install"
 
-# we are not starting the configserver service with the --configsvr
-# commandline option because right now this only changes the port it's
-# running on, and we are overwriting this port anyway.
+# mongodb_instance will set configsvr = true in the config file.
+# http://docs.mongodb.org/manual/reference/configuration-options/#sharded-cluster-options
+# we still explicitly set the port and small files.
```
Comment (josephholsten) [inline recipes/configserver.rb:28]: this comment isn't entirely true, `smallfiles` is set to true when `configsvr` is enabled at [src/mongo/db/db.cpp:1115](https://github.com/mongodb/mongo/blob/v2.4/src/mongo/db/db.cpp#L1115)

Comment (josephholsten) [inline recipes/configserver.rb:28]: oh, rereading the comment, I understand what it means. nevermind.

**Comment by josephholsten** [comment]
lgtm

**Comment by jamesonjlee** [comment]
i can't wait to fix this stuff.

File: recipes/configserver.rb (hunk 1 above)
Comment (jspiewak) [inline recipes/configserver.rb:28]: Could I re-word a little to be more clear?

**Pull request merged**